core/frontend/src/tile/OrbitGtTileTree.ts

```diff
@@ -280,7 +280,7 @@ export class OrbitGtTileTree extends TileTree {
     const tileCount = frameData.tilesToRender.size();
 
     // Inform TileAdmin about tiles we are handling ourselves...
-    IModelApp.tileAdmin.addExternalTilesForUser(args.context.viewport, { requested: frameData.tilesToLoad.size(), selected: tileCount, ready: tileCount });
+    IModelApp.tileAdmin.addExternalTilesForUser(args.context.viewport, { requested: frameData.tilesToLoad.size(), selected: tileCount, ready: tileCount, hasMissingData: frameData.hasMissingData() });

```

Comment:
I am realizing now that this could not be tracked properly if there were multiple external sources adding themselves to the external statistics - since this is a boolean flag it does not accumulate.  I should make it a count, or just "lie" about the requested number of tiles (bump it by one if this flag is toggled on).  Marking as draft as I reconsider.